application/main.js

```diff
@@ -19,8 +19,8 @@ function createWindow() {
         });
 
         mainWindow.loadFile(path.join(__dirname, 'web/index.html'));
-        //if (process.env.NODE_ENV === 'development')
-        // mainWindow.webContents.openDevTools();
+        // if (process.env.NODE_ENV === 'development')
+        //     mainWindow.webContents.openDevTools();
```

Comment:
is there a reason why we're commenting this out?

Reply:
It was originally commented out if you look at the file in the main branch, I just uncommented it for development purposes